### DISCUSSION TITLE
[Bug] the publicPath of monaco-editor-webpack-plugin not working

### DISCUSSION THREAD

**Discussion by KamiC6238** [microsoft/monaco-editor#3708] [Q&A]
i want to use worker from cdn, so i build my monaco web worker and upload to cdn server.

but when i start my local server and create monaco-editor, i get error from editor.worker.bundle.js 404.

my webpack.config.js

![image](https://user-images.githubusercontent.com/23523595/169988318-09d8985d-314d-477f-b468-b49d8a1b43ac.png)
![image](https://user-images.githubusercontent.com/23523595/169988385-f214a360-4104-473a-8f94-ba64490ed209.png)

i am appreciate if someone can tell me how to solve this problem..

**Comment by Ronchan0805** [comment]
![image](https://user-images.githubusercontent.com/74052977/170615215-8006e392-5efd-4684-90bf-c3a8824006f5.png)
Similarly, I had this problem...
